.idea/runConfigurations/Firestore_Integration_Tests__Firestore_Emulator_.xml

```diff
@@ -1,19 +1,18 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Firestore Integration Tests (Firestore Emulator)" type="AndroidTestRunConfigurationType" factoryName="Android Instrumented Tests">
     <module name="firebase-firestore" />
-    <option name="TESTING_TYPE" value="1" />
+    <option name="TESTING_TYPE" value="0" />
     <option name="INSTRUMENTATION_RUNNER_CLASS" value="" />
     <option name="METHOD_NAME" value="" />
     <option name="CLASS_NAME" value="" />
-    <option name="PACKAGE_NAME" value="com.google.firebase.firestore" />
```

Comment:
Removing the package qualification seems wrong (especially since you're adding it below).

Reply:
The change of `TESTING_TYPE` from `1` to `0` above reflects changing the "Testing Type" from "All in package" to "All in module" in the Android Studio UI.  The "All in Module" value will run all tests in the `androidTest` folder regardless of their Java package name.  As a result, there is no need to specify the Java package of the tests to execute, and hence the removal of this line.  To me, this change makes the configuration slightly more robust: since we want to run all tests I think it's better to do that than to specify to run all tests in a given Java package, which just happens to contain all of the tests.

Reply:
Also, in response to your comment about what this "fixes"... the change to this file does not fix anything.  It is merely a clean-up.